### PR TITLE
[WIP] i18n(import): stack pack translations

### PR DESCRIPTION
**Pull request by exterkamp**
**Summary**
Stack Pack translations are trickling in, will keep this updated as new translations come in.

### CODE REVIEW

**Comment by patrickhulce** [comment]
We should have the now.sh deployment show us an arabic report or something too :D

File: lighthouse-core/lib/i18n/locales/ar.json [hunk 1]
```diff
@@ -875,6 +875,24 @@
   "lighthouse-core/config/default-config.js | seoCategoryTitle": {
     "message": "تحسين محركات البحث"
   },
+  "lighthouse-core/config/default-config.js | seoContentGroupDescription": {
+    "message": "Format your HTML in a way that enables crawlers to better understand your app’s content."
```
Comment (patrickhulce) [inline lighthouse-core/lib/i18n/locales/ar.json]: they just didn't get to all of them?

Comment (patrickhulce) [reply lighthouse-core/lib/i18n/locales/ar.json]: oh I guess this is what "will keep this updated as new translations come in." was about :)

Comment (exterkamp) [reply lighthouse-core/lib/i18n/locales/ar.json]: Yeah, some languages aren't in yet :wink: , this is just a WIP as they come in.

**Pull request closed without merging**